.github/workflows/ci.yml

```diff
@@ -74,3 +74,73 @@ jobs:
         run: |
           aws s3 cp ${PROJECT}_${NEW_VERSION}.bin s3://dev-vitro-shard-applications/bumped-versions/
           aws s3 cp ${PROJECT}_${NEW_VERSION}_upgrade_package.swu s3://dev-vitro-core-sys-images/
+
+  build-telementry:
+    name: Build telementry test app
+    runs-on: self-hosted
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          path: "shard-v2-app-template"
+      - name: Get version
+        run: |
+          cd shard-v2-app-template
+          MAJOR=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 1p`
+          MINOR=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 2p`
+          PATCH=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 3p`
+          RC=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 4p`
+          CURRENT_VERSION="$MAJOR.$MINOR.$PATCH.$RC"
+          PROJECT="telementry-test"
```

Comment:
Also here, telemetry, please check other appearances

Reply:
Yes, I was convinced it was the other way around. Replaced all occurrences.